python/paddle/utils/code_gen/generate_op.py

```diff
@@ -65,6 +66,113 @@ def main(api_yaml_path, backward_yaml_path, output_op_path,
         backward_apis = [restruct_io(api) for api in backward_apis]
     backward_api_dict = to_named_dict(backward_apis)
 
+    with open(api_version_yaml_path, "rt") as f:
+        api_versions = yaml.safe_load(f)
+    # add api version info into api
+    for api_version in api_versions:
+        forward_api_dict[api_version['api']]['version'] = api_version['version']
```

Comment:
We can later move the parsing and validation of `api_version.yaml` and `args_compat.yaml` to `parse_api.py` and `cross_validate.py` to ensure that there is no error in the yaml before code genration.

Reply:
ok